drivers/storage/portworx/testspec/ccmGoPhonehomeConfigMap.yaml

```diff
@@ -12,7 +12,7 @@ data:
   ccm.properties: |-
     {
       "product_name": "portworx",
-      "port": "10090",
+      "port": "10024",
```

Comment:
I believe this would be startPort + 20 = 10021, assuming the start port for this test is 10001.
@jrivera-px I am assuming you are doing the same in porx.

Reply:
Start port + 20 yeah...

Reply:
why +20? the 9021 is in use seems https://github.com/portworx/porx/blob/master/build/piglet-ubuntu/docker-entry-point.sh#L2575

Reply:
It will come out to 9021 only if the start port is 9001. We are talking of a different start port. If the start port is not 9001, then we just do start port + 20.